src/selectors/columnWidths.js

```diff
@@ -55,21 +55,33 @@ function columnWidths(
     columnProps,
     viewportWidth
   );
-  const { fixedColumns, fixedRightColumns, scrollableColumns } =
-    groupColumns(newColumnProps);
+  const {
```

Comment:
The net change in this selector is that this now also calculates the width, offsets, and indexes of column groups (similar to normal columns).
This is really helpful because the APIs that we expose can simply use the selector to figure out column/columngroup details.